File: templates/component-pack/android/src/main/kotlin/myapp/example/MyAppExample.kt
// Component-pack template. Calls `register` from your host app
// alongside `WunFoundation.register(...)`.

package myapp.example

import wun.Registry

object MyAppExample {
    fun register(registry: Registry) {
        registry.register("myapp/MyComponent", MyComponentRenderer.render)
    }
}

File: templates/component-pack/ios/Sources/MyAppExample.swift
// Bootstraps `myapp/*` SwiftUI renderers into a Wun registry. Mirrors
// `WunFoundation.register(into:)`; the host app calls both in turn.

import Foundation
import Wun

public enum MyAppExample {
    public static func register(into registry: Registry) {
        registry.register("myapp/MyComponent", MyAppMyComponent.render)
    }
}
